### DISCUSSION TITLE
Responsiveness - Is it possible to implement it?

### DISCUSSION THREAD

**Discussion by Xavier9922** [KevinVandy/material-react-table#443] [Q&A]
Hi all, I'm new on the project. I was wondering if by any chance any of you have worked with responsiveness on the table for mobile, something like this: https://www.npmjs.com/package/react-super-responsive-table#demo 
Would it be possible to use a mix of Column Hiding and viewport alongside Sub-Rows Feature to get something similar as the above example? I don't think this counts as a feature request but if it is possible I might try to contribute to enable this
Any advice?

**Comment by MPatyn** [comment]
Did you solve this problem?
